kernel/enter.c

```diff
@@ -711,13 +707,13 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     /* Set the 'run-proc' which is called by the target to run new threads */
     ECHECK(myst_tcall_set_run_thread_function(myst_run_thread));
 
-#ifdef MYST_ENABLE_LEAK_CHECKER
-    /* print out memory statistics */
-    // myst_dump_malloc_stats();
-#endif
-
     myst_times_start();
 
+#if !defined(MYST_RELEASE)
```

Comment:
Should we check the debug mode of the application also? Without that, anybody can use a debug build of Mystikos to shell into the release build of an enclave application.

Reply:
I think we should discuss in person. I am confused about this.